src/test/java/com/exasol/adapter/sql/SqlLiteralIntervalTest.java

```diff
@@ -0,0 +1,45 @@
+package com.exasol.adapter.sql;
+
+import com.exasol.adapter.metadata.DataType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class SqlLiteralIntervalTest {
+    private static final String VALUE = "2019-02-07";
+    private SqlLiteralInterval sqlLiteralIntervalDayToSecond;
+    private SqlLiteralInterval sqlLiteralIntervalYearToMonth;
+    private DataType dayToSecond;
+    private DataType yearToMonth;
+
+    @BeforeEach
+    void setUp() {
+        dayToSecond = DataType.createIntervalDaySecond(1, 2);
+        yearToMonth = DataType.createIntervalYearMonth(3);
+        sqlLiteralIntervalDayToSecond = new SqlLiteralInterval(VALUE, dayToSecond);
+        sqlLiteralIntervalYearToMonth = new SqlLiteralInterval(VALUE, yearToMonth);
+    }
+
+    @Test
+    void testGetValue() {
+        assertThat(sqlLiteralIntervalDayToSecond.getValue(), equalTo(VALUE));
+    }
+
+    @Test
+    void testToSimpleSql() {
+        assertThat(sqlLiteralIntervalDayToSecond.toSimpleSql(), equalTo("INTERVAL '" + VALUE + "' DAY (1) TO SECOND (2)"));
```

Comment:
Please split into two tests.
Please also test with fractions of seconds. Parameterized JUnit 5 tests can make this more comfortable and extensible.

Reply:
done